js/slideshowcontrols.js

```diff
@@ -350,7 +350,9 @@
 		 * @private
 		 */
 		_next: function () {
-			Gallery.Share.hideDropDown();
+			if(!_.isUndefined(Gallery.Share)){
```

Comment:
I've just copied it from here... https://github.com/nextcloud/gallery/blob/41400d13581407675255ac2a172a0aa73eba1b0d/js/gallery.js#L340